lib/Common/DataStructures/quicksort.h

```diff
@@ -1,103 +0,0 @@
-//-------------------------------------------------------------------------------------------------------
-// Copyright (C) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
-//-------------------------------------------------------------------------------------------------------
-#pragma once
-
-namespace regex
-{
-    template <class T, bool partitionFromMiddle> class QuickSort
-    {
-    public:
-        static void Sort(T* low, T* high, Comparer<T> * comp)
```

Comment:
I don't know how qsort_s is implemented, but it is much smarter and faster than that basic quicksort implementation. I just did a quick test and even on release build this QuickSort crashes after ~22000 elements and it takes a few seconds to reach that point where as qsort_s terminates in no time at all